disperser/batcher/metrics.go

```diff
@@ -115,9 +122,37 @@ func NewMetrics(httpPort string, logger common.Logger) *Metrics {
 		),
 	}
 
+	finalizerMetrics := FinalizerMetrics{
+		Blobs: promauto.With(reg).NewCounterVec(
+			prometheus.CounterOpts{
+				Namespace: namespace,
+				Name:      "finalizer_num_blobs",
+				Help:      "number of blobs in each state",
+			},
+			[]string{"state"},
```

Comment:
A comment about what are the possible values for "state" will be helpful.

Same for the "stage" below for latency metric